### PR TITLE
Scaffolding for fixmynetboot

### PR DESCRIPTION
**Pull request by insomniacslk**
Example run:
```
$ go run .
#1 Running check 'wlp2s0 exists'.. OK
#2 Running check 'eth0 exists'.. failed: route ip+net: no such network interface
  -> running remediation
     Remediation for 'eth0 exists' failed: no trace of eth0 in dmesg
#3 Running check 'eth0 has link-local'.. OK
```

Signed-off-by: Andrea Barberio <insomniac@slackware.it>

### CODE REVIEW

File: cmds/fixmynetboot/dmesg.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import (
+	"strings"
+	"syscall"
+	"unsafe"
+)
+
+// shamelessly copied from u-root/cmds/dmesg
+const (
+	_SYSLOG_ACTION_READ_ALL = 3
```
Comment (pmazzini) [inline cmds/fixmynetboot/dmesg.go:11]: should we pay attention to the sticker ci here?

Comment (insomniacslk) [reply cmds/fixmynetboot/dmesg.go:11]: not really, this is a non-Go constant

Comment (insomniacslk) [reply cmds/fixmynetboot/dmesg.go:11]: https://elixir.bootlin.com/linux/v3.2/source/include/linux/syslog.h#L31

**Review by pmazzini — APPROVED** [review]
lgtm

**Pull request merged**